### DISCUSSION TITLE
Git Hub

### DISCUSSION THREAD

**Discussion by Suhas-30** [Cyfrin/foundry-full-course-cu#2256] [Q&A]
I can't push my code to github

**Comment by EngrPips** [comment]
What is the problem?

**Reply by Suhas-30** [reply]
git push -u origin main
Username for 'https://github.com': Suhas-30
Password for 'https://Suhas-30@github.com': 
remote: Permission to Suhas-30/Foundry-Fund-Me.git denied to Suhas-30.
fatal: unable to access 'https://github.com/Suhas-30/Foundry-Fund-Me.git/': The requested URL returned error: 403
suhas@DESKTOP-8DHC3LB:/mnt/c/Users/LENOVO/Documents/Foundry/Fund-ME$

**Reply by Suhas-30** [reply]
?

**Reply by EngrPips** [reply]
Hey check this stack overflow post https://stackoverflow.com/questions/17857283/permission-denied-error-on-github-push and see if it helps you resolve this issue.

**Comment by RianHodl** [comment]
I have the same issue, how to resolve this issue?

**Reply by EngrPips** [reply]
I replied in the discussion you opened.
